corporate/lib/stripe.py

```diff
@@ -753,6 +753,37 @@ def update_data_for_checkout_session_and_payment_intent(
     def org_name(self) -> str:
         pass
 
+    def get_past_invoices_session_url(self) -> str:
+        headline = "List of past invoices"
+        customer = self.get_customer()
+        assert customer is not None and customer.stripe_customer_id is not None
+
+        # Check if customer has any $0 invoices.
+        if stripe.Invoice.list(
+            customer=customer.stripe_customer_id,
+            limit=1,
+            status="paid",
+            total=0,
+        ).data:  # nocoverage
```

Comment:
`total` parameter was not documented but it works. https://stripe.com/docs/api/invoices/list
<img width="1607" alt="Screenshot 2024-02-03 at 10 59 34 AM" src="https://github.com/zulip/zulip/assets/25124304/212d2edb-9785-429f-8829-ada1bf04a89a">



<img width="1058" alt="Screenshot 2024-02-03 at 12 56 08 PM" src="https://github.com/zulip/zulip/assets/25124304/2ea9ce30-517f-4c7b-98c6-b0e17920521f">